ApplicationLibCode/ProjectDataModel/Summary/RimCsvUserData.cpp

```diff
@@ -24,6 +24,12 @@
 #include "RifCsvUserData.h"
 #include "RifKeywordVectorUserData.h"
 #include "RifSummaryReaderInterface.h"
+#include "RifSummaryReaderMultipleFiles.h"
+
+#include "RifMultipleSummaryReaders.h"
```

Comment:
Make sure all Rif-files are organized in same section

Reply:
Fixed.